### ISSUE TITLE
Search box for structure and minerals dialogs

### ISSUE THREAD

**Issue by ghuotvez** [NRCan/GSC-Field-Application#20]
Suggestion from Annick.

She had a blast from the litho search box. Having it for struc and mineral would greatly enhance user experience.

[https://gsc-datacollection.visualstudio.com/Mockup1/_workitems/edit/468](url)

**Comment by jameljoseph** [comment]
Working on this now!

**Comment by ghuotvez** [comment]
For behavior purposes, we can't keep mineral abbreviation and the full name while doing a search entry in the box. @jameljoseph has then replaced it with a normal behavior which is to show the field M_DICTIONARY.DESCRIPTION field instead of CODE, in which there is the abbreviation. This then implies a change on the field note page regarding the mineral cards. In them we used to showcase the abbreviation and the full name in parenthesis. This is no longer relevant and we'll drop this. I talked a bit about this with @jameljoseph and @eygirard and we'll go with this. Abbreviations were good back when notes were taken on paper and note books, but now there is not much argument to go with this.
